### PR TITLE
Feature/codeowners

### PR DESCRIPTION
**Pull request by 0x78f1935**
- Added Code Owner file
- Added Readme placeholder

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+                                    (____/                              
+
+
+---
+
+One of the first gamejames of Colour Coded!
+
+# Get started
+
+Say; Hello World :)
```
Comment (Visssarion) [inline README.MD:17]: BOO!!!! (just a test)

**Review by Visssarion — CHANGES_REQUESTED** [review]
test!!!

File: README.MD (hunk 1 above)
Comment (0x78f1935) [reply README.MD:17]: Very good test, please test review again :)

**Review by Visssarion — APPROVED** [review]
LGFM! 👍

**Review by HaiKonofanDesu — APPROVED** [review]

**Pull request merged**